contracts/strategies/BaseStrategy.sol

```diff
@@ -1,29 +1,29 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity 0.8.19;
 
+import "./IStrategy.sol";
+
 import {ERC20} from "@openzeppelin/token/ERC20/ERC20.sol";
```

Comment:
We are using both Solady and Openzeppelin for ERC20, 721, etc., should we stick to Solady? Seems we can do everything with that. Not sure about the upgradeable ERC 🤔 I think we still need OZ for that.

Reply:
Note sure if OZ is using 2612 yet for ERC20 also.